longest_common_substring.py

```diff
@@ -0,0 +1,15 @@
+def longest_common_substring(str_one: str, str_two: str) -> int:
+    len_one, len_two = len(str_one), len(str_two)
+    table = [[0] * (len_one + 1) for _ in range(len_two + 1)]
```

Comment:
I didn't understood why you used table

Reply:
just a random matrix to store the lengths

Reply:
Resolve the conflicts

Reply:
Conflict has been resolved, kindly check